src/MatBlazor/wwwroot/dist/matBlazor.js

```diff
@@ -1 +1 @@
ion(){this.unlisten("keydown",this.handleKeydown_),this.unlisten("transitionend",this.handleTransitionEnd_),this.list_&&this.list_.destroy();var t=d.MODAL;this.scrim_&&this.handleScrimClick_&&this.root.classList.contains(t)&&(this.scrim_.removeEventListener("click",this.handleScrimClick_),this.open=!1)},e.prototype.getDefaultFoundation=function(){var t=this,e={addClass:function(e){return t.root.classList.add(e)},removeClass:function(e){return t.root.classList.remove(e)},hasClass:function(e){return t.root.classList.contains(e)},elementHasClass:function(t,e){return t.classList.contains(e)},saveFocus:function(){return t.previousFocus_=document.activeElement},restoreFocus:function(){var e=t.previousFocus_;e&&e.focus&&t.root.contains(document.activeElement)&&e.focus()},focusActiveNavigationItem:function(){var e=t.root.querySelector("."+s.MDCListFoundation.cssClasses.LIST_ITEM_ACTIVATED_CLASS);e&&e.focus()},notifyClose:function(){return t.emit(f.CLOSE_EVENT,{},!0)},notifyOpen:function(){return t.emit(f.OPEN_EVENT,{},!0)},trapFocus:function(){return t.focusTrap_.trapFocus()},releaseFocus:function(){return t.focusTrap_.releaseFocus()}},n=d.DISMISSIBLE,i=d.MODAL;if(this.root.classList.contains(n))return new c.MDCDismissibleDrawerFoundation(e);if(this.root.classList.contains(i))return new u.MDCModalDrawerFoundation(e);throw new Error("MDCDrawer: Failed to instantiate component. Supported variants are "+n+" and "+i+".")},e}(r.MDCComponent);e.MDCDrawer=p},function(t,e,n){},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0}),e.init=function(){}},function(t,e,n){},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0}),e.init=function(t){t.matBlazorRef=new i.MDCLinearProgress(t)},e.setProgress=function(t,e){t.matBlazorRef.progress=e},e.setBuffer=function(t,e){t.matBlazorRef.buffer=e};var i=n(149)},function(t,e,n){"use strict";Object.defineProperty(e,"__esModule",{value:!0}),e.MDCLinearProgress=void 0;var i=n(0),r=n(2),o=n(150),a=function(t){function e(){return null!==t&&t.apply(this,arguments)||this}return(0,i.__extends)(e,t),e.attachTo=function(t){return new e(t)},Object.defineProperty(e.prototype,"determinate",{set:function(t){this.foundation.setDeterminate(t)},enumerable:!0,configurable:!0}),Object.defineProperty(e.prototype,"progress",{set:function(t){this.foundation.setProgress(t)},enumerable:!0,configurable:!0}),Object.defineProperty(e.prototype,"buffer",{set:function(t){this.foundation.setBuffer(t)},enumerable:!0,configurable:!0}),Object.defineProperty(e.prototype,"reverse",{set:function(t){this.foundation.setReverse(t)},enumerable:!0,configurable:!0}),e.prototype.open=function(){this.foundation.open()},e.prototype.close=function(){this.foundation.close()},e.prototype.getDefaultFoundation=function(){var t=this,e={addClass:function(e){t.root.classList.add(e)},forceLayout:function(){t.root.getBoundingClientRect()},setBufferBarStyle:function(e,n){var i=t.root.querySelector(o.MDCLinearProgressFoundation.string
```

Comment:
This is the true fix. There were 3 console.logs that aren't needed. I removed those.